server/src/infra/repositories/database.repository.ts

```diff
@@ -13,17 +13,20 @@ import { vectorExt } from '@app/infra/database.config';
 import { Injectable } from '@nestjs/common';
 import { InjectDataSource } from '@nestjs/typeorm';
 import AsyncLock from 'async-lock';
+import { Span } from 'nestjs-otel';
 import { DataSource, EntityManager, QueryRunner } from 'typeorm';
-import { isValidInteger } from '../infra.utils';
+import { DecorateAll, isValidInteger } from '../infra.utils';
 import { ImmichLogger } from '../logger';
 
+@DecorateAll(Span())
 @Injectable()
 export class DatabaseRepository implements IDatabaseRepository {
   private logger = new ImmichLogger(DatabaseRepository.name);
   readonly asyncLock = new AsyncLock();
 
   constructor(@InjectDataSource() private dataSource: DataSource) {}
 
+  @Span('db.get_extension_version')
```

Comment:
What does this do?

Reply:
Oh, I just added that while debugging and forgot about it haha